src/which.zig

```diff
@@ -133,7 +140,20 @@ fn whichWin(buf: *bun.WPathBuffer, path: []const u8, bin: []const u8) ?[:0]const
         return searchBin(buf, bin_utf16.len, check_windows_extensions);
     }
 
-    if (path.len == 0) return null;
+    // check if bin is in cwd
+    if (bun.strings.containsChar(bin, '/')) {
```

Comment:
I think these should be backslashes because this is the windows version. Or maybe we should accept both?

Reply:
it should be both